src/libraries/System.Net.Ping/src/System.Net.Ping.csproj

```diff
@@ -102,6 +102,7 @@
     <Reference Include="System.Threading.ThreadPool" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+    <Reference Include="System.Collections.Specialized" />
```

Comment:
Is this needed?

Reply:
seems like. I had same doubt and build fails without it.